tasks/token.yml

```diff
@@ -36,3 +38,32 @@
   set_fact:
     gitea_app_admin_token: '{{ gitea_app_admin_token_raw.content | b64decode | trim }}'
   when: gitea_token_file.stat.exists
+
+- name: Check if admin package replication API token file exists
+  stat:
+    path: '{{ gitea_service_path }}/replication-token'
+  register: gitea_replication_token_file
```

Comment:
Looks like we're just repeating the same stuff as above. Why can't we just re-use `tasks/token.yml` with `include_tasks` but just provide additional `vars` to create a different token?